src/app/pages/applications/forms/catalog-add-form.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import * as _ from 'lodash';
+import { FieldConfig } from '../../common/entity/entity-form/models/field-config.interface';
+import { FieldSet } from '../../common/entity/entity-form/models/fieldset.interface';
+import { ModalService } from '../../../services/modal.service';
+import { DialogService } from '../../../services/index';
+import  helptext  from '../../../helptext/apps/apps';
+import { EntityFormComponent } from 'app/pages/common/entity/entity-form';
+
+@Component({
+  selector: 'app-catalog-form',
+  template: `<entity-form [conf]="this"></entity-form>`
+})
+export class CatalogAddFormComponent {
+  protected queryCall: string = 'catalog.query';
+  protected customFilter: any[];
+  protected addCall: string = 'catalog.create';
+  protected isEntity: boolean = true;
+  protected entityForm: EntityFormComponent;
+  private title= helptext.catalogForm.title;
+  private dialogRef: any;
+  protected fieldConfig: FieldConfig[];
+  public fieldSets: FieldSet[] = [
+    {
+      name: 'Name',
+      width: '100%',
+      config: [
+        {
+          type: 'input',
+          name: 'label',
+          placeholder: helptext.catalogForm.name.placeholder,
+          tooltip: helptext.catalogForm.name.tooltip,
+          required: true,
+        },
+        {
+          type: 'checkbox',
+          name: 'force',
+          placeholder: helptext.catalogForm.forceCreate.placeholder,
+          tooltip: helptext.catalogForm.forceCreate.tooltip,
+          value: true,
```

Comment:
```suggestion
          type: 'checkbox',
          name: 'force',
          placeholder: helptext.catalogForm.forceCreate.placeholder,
          tooltip: helptext.catalogForm.forceCreate.tooltip,
          value: false,
```

easy editable version of the feedback from @sonicaj